.github/workflows/ci.yml

```diff
@@ -7,7 +7,10 @@ on:
       - main
     tags:
       - v*
+    schedule:
+      - cron: "0 0 * * 1-5"
```

Comment:
@joshwlewis I was wondering why the nightly images weren't being updated - turns out this YAML block is over-indented and so being ignored. (If I open this workflow file in VSCode, where I have the GitHub Actions extension enabled, I get a schema validation lint warning.)